### DISCUSSION TITLE
InvalidateQueries not working as expected

### DISCUSSION THREAD

**Discussion by Itzgaur** [TanStack/query#8189] [Q&A]
I am using mutations for login and logout functionalities in my application. However I am using invalidatequeries in both for authenticating the user. Invalidation is working fine in login but in logout it does not seems like working as expected.

**Comment by Itzgaur** [comment]
here is the codeSanbox for reference

https://codesandbox.io/p/github/Itzgaur/LinkedLikeApp/main?file=%2Ffrontend%2Fsrc%2FApp.jsx&import=true&workspaceId=302f336f-440a-4457-8953-d410c4ce4afa

https://codesandbox.io/p/github/Itzgaur/LinkedLikeApp/main?file=%2Ffrontend%2Fsrc%2Ffeatures%2Fauth%2FuseLogout.js&import=true&workspaceId=302f336f-440a-4457-8953-d410c4ce4afa

**Reply by TkDodo** [reply]
the sandboxes don’t work - likely they are private. Still, please be more precice. “does not work as expected” doesn’t tell me anything. What do you expect? What does it do instead?

**Reply by Itzgaur** [reply]
Never mind I found the issue. The issue was I was not  configuring it right.
